feature/amazon/src/main/java/com/revenuecat/purchases/amazon/storeProductConversions.kt

```diff
@@ -61,5 +54,80 @@ fun Product.toStoreProduct(marketplace: String): StoreProduct {
     )
 }
 
+internal fun String.extractPrice(
+    currency: Currency,
+    numberFormat: NumberFormat
+): Price {
+    debugLog("Received price is $this. Extracting currency and amount.")
```

Comment:
do we wanna leave all of these logs in, or maybe just the case where currency doesn't match locale?

Reply:
I think makes sense to do it only on error cases. I just realized I also need to extract the strings